core/storage/collection/gae_models.py

```diff
@@ -229,6 +229,28 @@ def _trusted_commit(
                     self.status == constants.ACTIVITY_STATUS_PRIVATE)
             ).put_async()
 
+    @classmethod
+    def export_data(cls, user_id):
+        """(Takeout) Export user-relevant properties of CollectionRightsModel.
+        Args:
+            user_id: str. The user_id denotes which user's data to extract.
+                If the user_id is not valid, this method returns None.
```

Comment:
Put this info in the Returns docstring instead. The typeinfo should be `dict or None`.

Reply:
done